File: app/src/main/java/wjy/yo/ereader/entity/Ordered.java
package wjy.yo.ereader.entity;

import java.util.Comparator;

public interface Ordered {

    Comparator<Ordered> Comparator = (o1, o2) -> (int) (o1.getNo() - o2.getNo());

    long getNo();

    void setNo(long no);
}

File: app/src/main/java/wjy/yo/ereader/entity/Ordered.kt
package wjy.yo.ereader.entity

import java.util.Comparator

interface Ordered {

    var no: Long

    companion object {

        val Comparator: Comparator<Ordered> =  Comparator { o1, o2 -> (o1.no-o2.no).toInt() }
    }
}
